sbin/send_email.py

```diff
@@ -132,14 +133,12 @@ def construct_mail_client():
 
 
 def get_email_queue(db):
-    """Get an active queue of emails waiting to be sent."""
-    result = db.execute(
-        "SELECT emails.id, emails.user_id, users.user_email, emails.subject," +
-        " emails.body FROM emails INNER JOIN users ON" +
-        " emails.user_id = users.user_id WHERE" +
-        " emails.sent is NULL AND emails.error = ''" +
-        " ORDER BY id LIMIT 100;")
-
+    """Get an active queue of internal emails waiting to be sent."""
+    query = """SELECT id, user_id, email_address, subject, body FROM emails
+    WHERE email_address SIMILAR TO '%%@(%%.{}|{})' AND sent is NULL AND
+    error = '' ORDER BY id LIMIT 100;"""
+    query = query.format(EMAIL_INTERNAL_FORMAT, EMAIL_INTERNAL_FORMAT)
```

Comment:
I will call it email internal domain. I think it's important to say that this is a different rate limit. No matter what I will need to do string checks on the email address to check if it belongs to the internal address domain. You are thinking it would be better to do these string checks in the PHP?

Reply:
I was suggesting to avoid doing any string comparisons, since the secondary email is its own property we could include a flag in the emails table to denote if the outgoing email is sending to a primary or secondary email.

Reply:
We can't necessarily assume that a primary address is internal. No matter what we will have to do string comparisons to see if we are sending to an internal or external address.

Reply:
I see, reading the issue I might of misunderstood the design goals and thought that secondary email implied external